double can't use _

```
java.lang.RuntimeException: java.lang.Exception: Property [ratePerSecond] with value [1_667] is not a double
	at com.hazelcast.simulator.coordinator.TestCaseRunner.awaitCompletion(TestCaseRunner.java:218)
	at com.hazelcast.simulator.coordinator.TestCaseRunner.invokeOnTargets(TestCaseRunner.java:191)
	at com.hazelcast.simulator.coordinator.TestCaseRunner.createTest(TestCaseRunner.java:185)
	at com.hazelcast.simulator.coordinator.TestCaseRunner.run0(TestCaseRunner.java:155)
	at com.hazelcast.simulator.coordinator.TestCaseRunner.run(TestCaseRunner.java:136)
	at com.hazelcast.simulator.coordinator.tasks.RunTestSuiteTask.runSequential(RunTestSuiteTask.java:181)
	at com.hazelcast.simulator.coordinator.tasks.RunTestSuiteTask.run0(RunTestSuiteTask.java:140)
	at com.hazelcast.simulator.coordinator.tasks.RunTestSuiteTask.run(RunTestSuiteTask.java:77)
	at com.hazelcast.simulator.coordinator.CoordinatorRunMonolith.run(CoordinatorRunMonolith.java:68)
	at com.hazelcast.simulator.coordinator.CoordinatorCli.run(CoordinatorCli.java:216)
	at com.hazelcast.simulator.coordinator.CoordinatorCli.main(CoordinatorCli.java:418)
Caused by: java.lang.Exception: Property [ratePerSecond] with value [1_667] is not a double
	at com.hazelca
```

--- a/simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java
+++ b/simulator/src/main/java/com/hazelcast/simulator/utils/PropertyBindingSupport.java
@@ -220,7 +220,7 @@ public final class PropertyBindingSupport {
         return true;
     }
 
-    private static String removeUnderscores(String s) {
+    public static String removeUnderscores(String s) {
         return s.replaceAll("_", "");
     }
 

--- a/simulator/src/main/java/com/hazelcast/simulator/worker/testcontainer/PropertyBinding.java
+++ b/simulator/src/main/java/com/hazelcast/simulator/worker/testcontainer/PropertyBinding.java
@@ -39,6 +39,7 @@ import static com.hazelcast.simulator.utils.AnnotationReflectionUtils.getProbeNa
 import static com.hazelcast.simulator.utils.AnnotationReflectionUtils.isPartOfTotalThroughput;
 import static com.hazelcast.simulator.utils.Preconditions.checkNotNull;
 import static com.hazelcast.simulator.utils.PropertyBindingSupport.bindAll;
+import static com.hazelcast.simulator.utils.PropertyBindingSupport.removeUnderscores;
 import static com.hazelcast.simulator.utils.ReflectionUtils.setFieldValue;
 import static java.lang.String.format;
 
@@ -132,7 +133,7 @@ public class PropertyBinding {
         }
 
         try {
-            return Integer.parseInt(value);
+            return Integer.parseInt(removeUnderscores(value));
         } catch (NumberFormatException e) {
             throw new IllegalTestException(format("Property [%s] with value [%s] is not an int", property, value));
         }
@@ -145,7 +146,7 @@ public class PropertyBinding {
         }
 
         try {
-            return Long.parseLong(value);
+            return Long.parseLong(removeUnderscores(value));
         } catch (NumberFormatException e) {
             throw new IllegalTestException(format("Property [%s] with value [%s] is not an long", property, value));
         }
@@ -158,7 +159,7 @@ public class PropertyBinding {
         }
 
         try {
-            return Double.parseDouble(value);
+            return Double.parseDouble(removeUnderscores(value));
         } catch (NumberFormatException e) {
             throw new IllegalTestException(format("Property [%s] with value [%s] is not a double", property, value));
         }

--- a/simulator/src/test/java/com/hazelcast/simulator/worker/testcontainer/PropertyBindingTest.java
+++ b/simulator/src/test/java/com/hazelcast/simulator/worker/testcontainer/PropertyBindingTest.java
@@ -41,6 +41,41 @@ public class PropertyBindingTest {
     }
 
     @Test
+    public void loadAsDouble_existing_wihUnderscores() {
+        TestCase testCase = new TestCase("foo")
+                .setProperty("doubleValue", "5_0d");
+        PropertyBinding binding = new PropertyBinding(testCase);
+
+        assertEquals(50, binding.loadAsDouble("doubleValue", 10), 0.1);
+    }
+
+    @Test
+    public void loadAsLong_nonExisting() {
+        TestCase testCase = new TestCase("foo");
+        PropertyBinding binding = new PropertyBinding(testCase);
+
+        assertEquals(10, binding.loadAsLong("longValue", 10));
+    }
+
+    @Test
+    public void loadAsLong_existing() {
+        TestCase testCase = new TestCase("foo")
+                .setProperty("longValue", 50);
+        PropertyBinding binding = new PropertyBinding(testCase);
+
+        assertEquals(50, binding.loadAsLong("longValue", 10));
+    }
+
+    @Test
+    public void loadAsLong_existing_withUnderscores() {
+        TestCase testCase = new TestCase("foo")
+                .setProperty("longValue", "5_0");
+        PropertyBinding binding = new PropertyBinding(testCase);
+
+        assertEquals(50, binding.loadAsLong("longValue", 10));
+    }
+
+    @Test
     public void loadAsInt_nonExisting() {
         TestCase testCase = new TestCase("foo");
         PropertyBinding binding = new PropertyBinding(testCase);
@@ -52,6 +87,15 @@ public class PropertyBindingTest {
     public void loadAsInt_existing() {
         TestCase testCase = new TestCase("foo")
                 .setProperty("intValue", 50);
+        PropertyBinding binding = new PropertyBinding(testCase);
+
+        assertEquals(50, binding.loadAsInt("intValue", 10));
+    }
+
+    @Test
+    public void loadAsInt_existing_withUnderscores() {
+        TestCase testCase = new TestCase("foo")
+                .setProperty("intValue", "5_0");
         PropertyBinding binding = new PropertyBinding(testCase);
 
         assertEquals(50, binding.loadAsInt("intValue", 10));

